docs-config.yaml

```diff
@@ -0,0 +1,42 @@
+port: 3000
+host: "localhost"
+uploads:
+  driver: "local"
+  local:
+    path: "./uploads"
+password: secretpassword
```

Comment:
this config should be ignored, or the password field shouldn't be there

Reply:
we need this password because you can't edit docs without being authenticated
this password is not secret data